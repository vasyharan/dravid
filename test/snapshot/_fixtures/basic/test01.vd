def foo(a, b, c) {
   (a + 10) * b - 1 / c
}

def main() {
  foo(1, 2, 3)
}