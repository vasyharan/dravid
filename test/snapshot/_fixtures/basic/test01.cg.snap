; ModuleID = 'my cool jit'
source_filename = "my cool jit"

define i64 @foo(i64 %a, i64 %b, i64 %c) {
entry:
  %addtmp = add i64 %a, 10
  %multmp = mul i64 %addtmp, %b
  %divtmp = sdiv exact i64 1, %c
  %subtmp = sub i64 %multmp, %divtmp
  ret i64 %subtmp
}

define i64 @main() {
entry:
  %calltmp = call i64 @foo(i64 1, i64 2, i64 3)
  ret i64 %calltmp
}
